[video_player] [iOS] Fixed a memory leak in the video player plugin. (#1660)
flutter/plugins: packages/video_player/CHANGELOG.md

## 0.10.1+5

> * [iOS] Fixed a memory leak in the video player plugin.

Reviewer: ```suggestion
* [iOS] Fixed a memory leak with notification observing.
```
